src/dualscreen.tsx

```diff
@@ -230,6 +233,20 @@ export class DualScreen extends BasePlugin<DualScreenConfig> implements IEngineD
     return thumbs[0] && thumbs[1] ? thumbs : thumbs[0] ?? thumbs[1];
   };
 
+  private _changeQuality = (label: string) => {
+    this._dualScreenPlayers.forEach(dualScreenPlayer => {
+      if(dualScreenPlayer.id != 'mainPlayer' && dualScreenPlayer.id != 'imagePlayer'){
```

Comment:
1. use prettier.
there should be space between `if` and `(`, same between `)` and `{`
2. use strict compare `!==` that requires less resources
3. use enums
`IMAGE_PLAYER_ID` for 'imagePlayer'
`MAIN_PLAYER_ID` for 'mainPlayer'